src/cli.ts

```diff
@@ -80,19 +80,22 @@ program
       Object.keys(reporters)
     )
   )
-  .option('--inline', 'Run inline journeys from heartbeat')
+  .option('--inline', 'read journeys from stdin instead of reading from files')
   .option('-r, --require <modules...>', 'module(s) to preload')
   .option('--sandbox', 'enable chromium sand-boxing')
-  .option('--rich-events', 'Mimics a heartbeat run')
-  .option('--no-headless', 'run browser in head-full mode')
+  .option(
+    '--rich-events',
+    'preset flag used when running monitors directly via Heartbeat'
+  )
+  .option('--no-headless', 'run with the browser in head-full mode')
   .option(
     '--capability <features...>',
     'Enable capabilities through feature flags'
   )
   .addOption(
     new Option(
       '--screenshots [flag]',
-      'take screenshots at end of each step'
+      'Control whether or not to capture screenshots at the end of each step'
```

Comment:
```suggestion
      'Control whether to capture screenshots at the end of each step'
```

`whether` does all the heavy lifting, no need to add more conditional wording.